.github/workflows/maven.yml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: GitHub CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'main'
+  pull_request:
+
+jobs:
+  build:
+    name: Maven Build CI
+    uses: jetty/.github/.github/workflows/maven-ci.yml@main
```

Comment:
Github advises that you use a release version here.
Otherwise maven caching (of dependencies) is broken.
So that means you have to create (at least) a tag.
This means each of these become something like 
`jetty/.github/.github/workflows/maven-ci.yml@5`

Reply:
Nah maven caching is based on hash of pom.xml https://github.com/jetty/jetty-build-support/pull/1#discussion_r1384247263